velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -144,13 +144,16 @@ class ServerResponse {
 } // namespace
 
 PrestoQueryRunner::PrestoQueryRunner(
+    memory::MemoryPool* rootPool,
     std::string coordinatorUri,
     std::string user,
     std::chrono::milliseconds timeout)
-    : coordinatorUri_{std::move(coordinatorUri)},
+    : ReferenceQueryRunner(rootPool),
+      coordinatorUri_{std::move(coordinatorUri)},
       user_{std::move(user)},
       timeout_(timeout) {
   eventBaseThread_.start("PrestoQueryRunner");
+  pool_ = ReferenceQueryRunner::rootPool()->addLeafChild("leaf");
```

Comment:
Is ReferenceQueryRunner:: needed?

Reply:
The function name was duplicate with the variable name `rootPool`. Removing `ReferenceQueryRunner::` and changing the variable name.